src/css/global.css

```diff
@@ -40,59 +33,89 @@ p{
   --FONT_SIZE_20: 20px;
   --FONT_SIZE_24: 24px;
   --FONT_SIZE_30: 30px;
+  --FONT_SIZE_36: 36px;
+  --FONT_SIZE_44: 44px;
   --FONT_SIZE_48: 48px;
   --FONT_SIZE_50: 50px;
   --FONT_SIZE_72: 72px;
-  --FONT_SIZE_44: 44px;
 
   --FONT_WEIGHT_700: 700;
 }
 
-@font-face {
+html {
+  font-family: Georgia, 'Times New Roman', Times, serif;
+}
+
+p, a, input, button, span {
   font-family: 'Axiforma';
-  src: url(../fonts/Axiforma-Regular.ttf);
-  font-display: swap;
-  font-style: normal;
+  letter-spacing: 0.02em;
 }
 
-.black-txt{
-color: var(--COLOR_BLACK);
+p {
+  color: var(--COLOR_DARK);
 }
 
-.white-txt{
-  color: var(--COLOR_WHITE);
+a {
+  text-decoration: none;
+  color: var(--COLOR_BLACK);
+}
+
+input {
+  min-height: 50px;
+  border-radius: 10px;
+}
+
+button{
+  cursor: pointer;
+}
+
+button:hover{
+  background-color: var(--COLOR_DARK);
+  transition: 0.5s;
+}
+
+header {
+  padding: 50px 10px 125px 10px;
 }
 
-.container{
+.container {
   position: relative;
   max-width: 1140px;
   margin: 0 auto;
 }
 
-.title{
-  font-size: var(--FONT_SIZE_48);
-  font-weight: 700;
+.logo {
+  font-family: Georgia, 'Times New Roman', Times, serif;
+  font-weight: var(--FONT_WEIGHT_700);
+  font-size: var(--FONT_SIZE_50);
+  line-height: 57px;
 }
 
+.black-txt {
+  color: var(--COLOR_BLACK);
+}
 
+.white-txt {
+  color: var(--COLOR_WHITE);
+}
 
-a{
-  font-family: 'Axiforma';
-  text-decoration: none;
-  color: var(--COLOR_BLACK);
+.title {
+  font-size: var(--FONT_SIZE_48);
+  font-weight: 700;
+  line-height: 95px;
 }
 
-.decor-orange{
+.decor-orange {
   color:var(--COLOR_ORANGE);
 }
 
-.hover-orange:hover{
+.hover-orange:hover {
   cursor: pointer;
   color:var(--COLOR_ORANGE);
   transition: 0.3s;
```

Comment:
`transition` and `cursor` should be on the `.hover_orange` class.

Reply:
I didn’t understand what needs to be corrected, transition and cursor on the .hover_orange class.